index.js

```diff
@@ -77,23 +78,32 @@ var run = function bramble (args) {
                         packagesToUpdate.push(pack[1] + '@' + semVerRange + pack[4]);
                         previousPackages.push(pack[1] + '@' + semVerRange + pack[2]);
                     });
-
-                    console.log(('Installing the latest packages ' + packagesToUpdate.toString().replace(',', ', ')).green);
-
+                    var installSpinner = new Spinner(('Installing the latest packages ' + packagesToUpdate.toString().replace(',', ', ') + ' %s').green);
+                    installSpinner.setSpinnerString(5);
+                    installSpinner.start();
                     return npm.install(packagesToUpdate, installOptions)
                         .then(function () {
+                            installSpinner.stop();
                             if (args.test) {
                                 return npm.test();
                             } else {
                                 return new Q();
                             }
                         }, function (err) {
+                            installSpinner.stop();
                             console.log(('NPM install failed').red);
                             throw err;
                         })
                         .fail(function () {
-                            console.log(('NPM tests failed restoring to old versions ' + previousPackages.toString().replace(',', ', ')).red);
-                            return npm.install(previousPackages, installOptions).fail(function (err) { throw err; });
+                            installSpinner.stop();
```

Comment:
I knew you were gonna say that. You're like the voice in the back of my head :speak_no_evil: